deploy/static/trivy-operator.yaml

```diff
@@ -2992,9 +2992,9 @@ metadata:
     app.kubernetes.io/managed-by: kubectl
 data:
   OPERATOR_LOG_DEV_MODE: "false"
-  OPERATOR_SCAN_JOB_TTL: ""
+  OPERATOR_SCAN_JOB_TTL: "30s"
   OPERATOR_SCAN_JOB_TIMEOUT: "5m"
-  OPERATOR_CONCURRENT_SCAN_JOBS_LIMIT: "10"
+  OPERATOR_CONCURRENT_SCAN_JOBS_LIMIT: "20"
```

Comment:
I understand you want to use this file for tests but it is auto generated so any changes here will not persist when generation takes place.

Instead of change the defaults as that is not the point of this PR, how about we test it differently? I found this unit test that can allow the modification for TTL values and be used to test this change https://github.com/aquasecurity/trivy-operator/blob/29d14bf6c8b5473d1a8c026c574cff2fdfcf22a2/pkg/operator/ttl_report_test.go#L28

Reply:
Thanks for the review and for the hint to the ttl_report_test. I added a test similar to that one in a new file [scanjob_test.go](https://github.com/aquasecurity/trivy-operator/pull/2375/files#diff-a4718cd1e513c622d28b559ef021936c1b851483ec55919ee0786f712cb3070d). It contains a test for both scenarios; When the job should be deleted immediately and when not. So even better than the env tests.
What do you think about that ? I could delete the env test in facvor of this test then.